testing/test_skipping.py

```diff
@@ -1444,3 +1444,70 @@ def test_pass():
     result.stdout.fnmatch_lines(
         ["SKIPPED [[]1[]] tests/test_1.py:2: unconditional skip"]
     )
+
+
+def test_skip_with_msg_is_deprecated(pytester: Pytester) -> None:
```

Comment:
Let's move these tests to `testing/deprecated_test.py`, and group them in a class to make their relation explicit:

```python
class TestSkipMsgArgumentDeprecated:

    def test_skip_with_msg_is_deprecated(self, pytester: Pytester) -> None:
        ...
```